File: python/challenge_2.py
#Challenge 2 (use python 2) ini adalah membuat output seperti berikut:
# 1 2 3 4 5 6 
# 1 2 3 4 5 6 
# 1 2 3 4 5 6 
# 1 2 3 4 5 6  

high = input("Tinggi : ")
wide = input("Lebar : ")

while (high > 0):
	for i in range(wide):
		print(i+1),

	print("")
	high -= 1

File: PHP/challenge_2.php
<?php
/*
Output:

1 2 3 4 5 6
1 2 3 4 5 6
1 2 3 4 5 6
1 2 3 4 5 6

*/
echo "Lebar: ";
$lebar = trim(fgets(STDIN));

echo "Tinggi: ";
$tinggi = trim(fgets(STDIN));

for($t = 0; $t < $tinggi; $t++){
    for($l = 1; $l <= $lebar; $l++){
        echo $l." ";
    }

    echo"\n";
}
echo"\n";
?>